code/demo/android-native/app/build.gradle

```diff
@@ -1,37 +1,59 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+  buildToolsVersion = '30.0.2'
+  compileSdkVersion sdkVersion.toInteger()
 
-    defaultConfig {
-        applicationId "org.orxproject.orxtestnative"
-        minSdkVersion 19
-        targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
+  ndkVersion "21.3.6528147"
 
-        ndk{
-            abiFilters "x86", "armeabi-v7a", "arm64-v8a", "x86_64"
-        }
+  defaultConfig {
+    applicationId "org.orxproject.orxtestnative"
+    targetSdkVersion sdkVersion.toInteger()
```

Comment:
targetSdk and compileSdk are not necessarily the same, this might introduce confusion for new users

Reply:
Not necessarily, but ideally yes. So let's enforce the ideal configuration (read about it [here](https://medium.com/androiddevelopers/picking-your-compilesdkversion-minsdkversion-targetsdkversion-a098a0341ebd#.bpqsg4ili)). Especially for an `orx` app where you rarely, if ever, need to care about these special settings.